src/environment.ts

```diff
@@ -147,14 +148,17 @@ export class SecureEnvironment implements MembraneBroker {
     remap(redValue: RedValue, blueValue: BlueValue, blueDescriptors?: PropertyDescriptorMap) {
         this.setRefMapEntries(redValue, blueValue);
         if (!isUndefined(blueDescriptors)) {
-            for (const key in blueDescriptors) {
+            const keys = ownKeys(blueDescriptors);
```

Comment:
I'm probably ok with this change for now... but remap is a public API IIRC, which up to this point supported elements from proto chain as well with the for in... but now it only supports own properties.

Reply:
I think the iteration is on a collection of descriptors we get using `getOwnPropertyDescriptors` which shouldn't impact the proto chain since the passed `blueDescriptors` only inherits from Object.